client/src/charts/canada/canada.js

```diff
@@ -197,23 +195,13 @@ export class Canada extends React.Component {
               checkBoxProps: { isSolidBox: true },
             }}
           />
-          <div
-            className="standard-legend-container"
-            style={{
-              maxHeight: "400px",
-              width: "100%",
-              overflowY: "hidden",
-              marginTop: "10px",
-            }}
-          >
-            <CanadaGraphBarLegend
-              prov={prov}
-              data={data}
-              alt_totals_by_year={alt_totals_by_year}
-              years={years}
-              formatter={formatter}
-            />
-          </div>
```

Comment:
This should remove the double container, I must have overlooked the checkbox comment earlier. Sorry for having to repeat yourself.